content/release.rst

```diff
@@ -86,23 +86,26 @@ Some functions of the **release manager** are:
   to make sure no regressions were introduced on the tests executed post-merge.
 * Creating high priority tickets in case the tests are failing on master.
 * Coordinating story tickets for the milestone.
-
+* After sending the release branch to RQM, check that the Downloads page is updated.
+* After the Downloads page is updated, ensure that the release branch is merged
+  with the master branch.
```

Comment:
and also check that a tag is created for the release, and the tag points to the release branch and not the release merge :)

Reply:
Done. 
Also adding:
``* Check that RQM has closed the release PR and associated Trac.``